Makefile

```diff
@@ -349,6 +349,11 @@ e2e-helm-deploy:
 		--set enableSecretRotation=true \
 		--set rotationPollInterval=30s
 
+.PHONY: e2e-helm-upgrade
+e2e-helm-upgrade:
+	helm upgrade csi-secrets-store manifest_staging/charts/secrets-store-csi-driver --namespace kube-system --reuse-values --timeout=15m -v=5 --debug --set filteredWatchSecret=true
```

Comment:
Shouldn't we use the e2e image tag and set the linux and windows images here? `--reuse-values` without that will use the release image.

Reply:
ahh.. Good catch..